ocrolib/hocr.py

```diff
@@ -14,8 +14,8 @@
 <head>
 <title>OCR Results</title>
 <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-<meta name="Description" content="OCRopus Output" />
-<meta name="ocr-system" content="ocropus-0.4" />
+<meta name="Description" content="ocropy Output" />
```

Comment:
Do we need the `Description` here? Is this the same information as the `title`? Should this be `DC`-prefixed?

--> My suggestion is to simply delete this `meta`-tag with `Description`. Also Tesseract does not have such a line.